.reek.yml

```diff
@@ -81,3 +81,4 @@ exclude_paths:
   - bin/
   - vendor/
   - db/migrate
+  - app/controllers/graphql_controller.rb
```

Comment:
Why are we excluding?

Reply:
generated file, from graphql gem, I dont want to refactor it to meet reek criterias